Mathlib/Algebra/Ring/Semireal/Defs.lean

```diff
@@ -0,0 +1,115 @@
+/-
+Copyright (c) 2024 Florent Schaffhauser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Florent Schaffhauser.
+-/
+
+import Mathlib.Algebra.Ring.Defs
```

Comment:
```suggestion
Authors: Florent Schaffhauser
-/
import Mathlib.Algebra.Ring.Defs
```
TBH I am not sure what is the problem here. At worst just copy/paste from another file and change your name (The VS Code extension can also produce it for you IIRC)

Reply:
Ah, it's the dot at the end of the line

Reply:
OK, I'll try again

Reply:
The formatting seems correct now :+1: